src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -1021,6 +1049,30 @@ public function iShouldBeAbleToGoToTheShippingStepAgain()
     }
 
     /**
+     * @Then I should be able to go to the complete step again
+     */
+    public function iShouldBeAbleToGoToTheCompleteStepAgain()
+    {
+        Assert::true(
+            $this->completePage->isOpen(),
+            'Checkout complete step should be opened, but it is not.'
+        );
+    }
+
+    /**
+     * @Then I should not be able to go to the payment step 
+     */
+    public function iShouldNotBeAbleToGoToThePaymentStep()
+    {
+        $this->selectShippingPage->nextStep();
+
+        Assert::false(
+            $this->selectPaymentPage->isOpen(),
+            'Checkout payment step should not be opened, but it is.'
```

Comment:
So if it return true then the open page will be payment page and the message is correct, because it shouldn't be open.